scripts/ingests/21Legg_ingest.py

```diff
@@ -0,0 +1,449 @@
+                'Luhman_2012': 'Luhm12.135',
+                'Luhman_2014': 'Luhm14.18',
+                'Mace_2013a': 'Mace13.6',
+                'Mace_2013b': 'Mace13.36',
+                'Meisner_2020a': 'Meis20.74',
+                'Meisner_2020b': 'Meis20.123',
+                'Pinfield_2014a': 'Pinf14.1009',
+                'Pinfield_2014b': 'Pinf14.1931',
+                'Pinfield_Gromadzki_2014': 'Pinf14.priv',
+                'Scholz_2010a': 'Scho10.8',
+                'Scholz_2010b': 'Scho10.92',
+                'Tinney_2005': 'Tinn05.2326',
+                'Warren_2007': 'Warr07.1400'
+                }
+
+    # convert reference names
+    first_ref = reference.split()[0]
+    # print('Leggett Ref: ', first_ref)
+    for leggett_ref, simple_ref in ref_dict.items():
+        if first_ref == leggett_ref:
+            first_ref = first_ref.replace(leggett_ref, simple_ref)
+            # print('Replaced: ', first_ref)
+
+    simple_search = find_publication(db, first_ref)
+    if simple_search[0]:
+        simple_ref = simple_search[1]
+    else:
+        msg = f'More than SIMPLE ref found for {first_ref}'
+        raise SimpleError(msg)
+    # print('Simple Ref: ', simple_ref)
+    return simple_ref
+
+
+if RECREATE_DB:
+    add_publications(db)
+
+legg21_table = ascii.read('scripts/ingests/apjac0cfet10_mrt.txt', format='mrt')
+
+primary_list = ['WISE J014656.66+423410.0A',
+                'WISEPA J045853.89+643452.9A',
+                "WISEPC J121756.91+162640.2A",
+                "2MASS J122554.32-273946.6A",
+                "CFBDSIR J145829.00+101343.0A",
+                '2MASSI J155302.20+153236.0A',
+                "WISEPA J171104.60+350036.8A"
+                ]
+
+binary_list = [
+    "WISE J014656.66+423410.0B",
+    "WISEPA J045853.89+643452.9B",
+    "WISEPC J121756.91+162640.2B",
+    "2MASS J122554.32-273946.6B",
+    "CFBDS J145829.00+101343.0B",
+    "2MASSI J155302.20+153236.0B",
+    "WISEPA J171104.60+350036.8B"
+]
+
+source_strings = []
+discovery_refs = []
+ras = []
+decs = []
+other_references = []
+# spectral type variables
+spt_source_strings = []
+sp_types = []
+sp_type_refs = []
+# spitzer phot variables
+iraci1_source_strings = []
+iraci2_source_strings = []
+iraci3_source_strings = []
+iraci4_source_strings = []
+iraci1_mags = []
+iraci2_mags = []
+iraci3_mags = []
+iraci4_mags = []
+iraci1_mags_err = []
+iraci2_mags_err = []
+iraci3_mags_err = []
+iraci4_mags_err = []
+iraci1_refs = []
+iraci2_refs = []
+iraci3_refs = []
+iraci4_refs = []
+
+for row, source in enumerate(legg21_table):
+    if not ma.is_masked(source['CBin']):
+        source_string = f"{source['Survey']} J{source['RA']}{source['Decl.']}{source['CBin']}"
+    else:
+        source_string = f"{source['Survey']} J{source['RA']}{source['Decl.']}"
+    source_strings.append(source_string)
+
+    ra, dec = convert_radec_to_decimal(source)
+    ras.append(ra)
+    decs.append(dec)
+    print(row, source_string, ra, dec)
```

Comment:
Minor thing but I prefer a standard iterator like `ijk` rather than `row` because enumerate is always returning an iterated integer as the first variable. `row` to me, means an an object like the physical row of the table.
```suggestion
for i, source in enumerate(legg21_table):
    if not ma.is_masked(source['CBin']):
        source_string = f"{source['Survey']} J{source['RA']}{source['Decl.']}{source['CBin']}"
    else:
        source_string = f"{source['Survey']} J{source['RA']}{source['Decl.']}"
    source_strings.append(source_string)

    ra, dec = convert_radec_to_decimal(source)
    ras.append(ra)
    decs.append(dec)
    print(i, source_string, ra, dec)
```

Reply:
I see what your saying. How about `row_number`?

Reply:
`row_number` or `rown` would also read nice